raiden/ui/cli.py

```diff
@@ -493,3 +497,26 @@ def append_report(subject, data):
     else:
         print('[5/5] smoketest had errors, report was written to {}'.format(report_file))
         sys.exit(1)
+
+
+@run.command()
+@OPTION_ADDRESS
+@OPTION_DATADIR
+def removedb(address, datadir):
```

Comment:
I'm no fan of duplicating options between the main- and subcommands.
I would suggest to only have the `--datadir` and `--address` options on the main command. 
I.e.
The way you've build it now the usage would be:
```
raiden removedb --datadir /bla
```
However (confusingly) the following wouldn't do the correct thing yet produce no error (as in delete the default dir not the specified one):
```
raiden --datadir /bla removedb
```
And yet more confusing this would also be possible:
```
raiden --datadir /foo removedb --datadir /bla
```

From my perspective the second would the "correct" usage but I'd like to get @LefterisJP @konradkonrad and @hackaugusto opinions on this.